src/tree/EnvironmentTreeItem.ts

```diff
@@ -58,8 +62,9 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
         return this.data.properties.buildId === 'default' ? productionEnvironmentName : `#${this.name} - ${this.data.properties.pullRequestTitle}`;
     }
 
-    public get description(): string | undefined {
-        return this.data.properties.sourceBranch;
+    public get description(): string {
+        const linkedTag: string = this.inWorkspace ? localize('linkedTag', '(linked)') : '';
```

Comment:
Not quite - you need to put the localized content in context whenever possible. Depending on the language, the ordering of the words might change.

localize('linkedTag', '{0} (linked)', branch)